easybuild/easyblocks/m/mxnet.py

```diff
@@ -97,21 +101,26 @@ def extract_step(self):
 
         for srcdir in [d for d in os.listdir(self.builddir) if d != os.path.basename(self.mxnet_src_dir)]:
             submodule, _, _ = srcdir.rpartition('-')
-            newdir = os.path.join(self.mxnet_src_dir, submodule)
+
+            # MXNet assumes the old name for oneDNN which was mkldnn
+            if submodule == 'oneDNN':
+                submodule = 'mkldnn'
+                # rename the file to 'mkldnn':
+                old_srcdir = srcdir
+                srcdir = srcdir.replace('oneDNN', 'mkldnn')
+                os.rename(os.path.join(self.builddir, old_srcdir), os.path.join(self.builddir, srcdir))
+
             olddir = os.path.join(self.builddir, srcdir)
+            newdir = os.path.join(self.mxnet_src_dir, '3rdparty', submodule)
```

Comment:
Does we also need the `1.0` version condition here w.r.t. using `3rdparty` or not?

Reply:
solved